tsconfig.json

```diff
@@ -2,13 +2,14 @@
   // Just make IDE happy
   "extends": "@silverhand/ts-config-react/tsconfig.base",
   "compilerOptions": {
-    "baseUrl": "./",
+    "baseUrl": ".",
```

Comment:
These changes are not relevant to PR title?

Reply:
split to another PR